Upstream-Status: Inappropriate [embedded specific]

Replace the OECORE_KNOWN_INTERPRETER_NAMES with the value of 
variable EGLIBC_KNOWN_INTERPRETER_NAMES.

Lianhao Lu, 08/01/2011

--- libc/elf/readlib.c.orig	2011-08-12 17:05:51.864470837 +0800
+++ libc/elf/readlib.c	2011-08-12 17:06:39.346942074 +0800
@@ -52,6 +52,7 @@
 #ifdef SYSDEP_KNOWN_INTERPRETER_NAMES
   SYSDEP_KNOWN_INTERPRETER_NAMES
 #endif
+  OECORE_KNOWN_INTERPRETER_NAMES
 };
 
 static struct known_names known_libs[] =
